tests/linkediting.js

```diff
@@ -73,8 +87,8 @@ describe( 'LinkEditing', () => {
 		} );
 
 		it( 'should be bound to th `linkHref` attribute (RTL)', () => {
-			return VirtualTestEditor
-				.create( {
+			return ClassicTestEditor
```

Comment:
Might be a follow-up but AFAICS the main `beforeEach()` editor was not destroyed and this looks like we could have two editors there.

Reply:
Adding `element` in tests and using `ClassicTestEditor` might create some memory leaks if multiple editors are created without removing previous one (from global `beforeEach()` ).